spi/identity-hub-spi/src/main/java/org/eclipse/edc/identityhub/spi/credentials/transformer/CredentialEnvelopeTransformer.java

```diff
@@ -22,7 +22,7 @@
  * The {@link MessageProcessor} is able to process messages that could contain data. If data available in the current implementation
  * we want to be sure that the data is in the correct format based on the Media type.
  */
-public interface CredentialEnvelopeTransformer<T extends CredentialEnvelope> {
+public interface CredentialEnvelopeTransformer<T> {
```

Comment:
See comment above: we should have a unified envelope for both VP and VC

Reply:
Using same envelope now for both vp and vc with a list of credentials